jazzhands/__main__.py

```diff
@@ -122,9 +128,15 @@ def main(argv):
         DO_COLLECT = DO_BUILD = True
     elif sys.argv[1] == 'setup':
```

Comment:
Undocumented command.

There's also an undocumented option, `--auto-npm`, that I'm not sure is a great idea

Reply:
Adding a warning for now, and I have gotten other feedback like this already... (it originally wasn't optional) so maybe it'll go away but I really like ti and want to save it still :-(